.jenkins/docker_cuda_theano/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM nvidia/cuda:8.0-cudnn5-devel-ubuntu16.04
```

Comment:
I did at first but got warning cudnn 6 is too new. Should we wait until wrapping and warnings are done, or we want to test now?